### ISSUE TITLE
Clarify README steps 

### ISSUE THREAD

**Issue by gitressa** [valthebald/ddev-matomo#4]
I followed the steps in the README, but can't get past `3. Database Setup` where I get a `Could not connect to the database server: Database access denied ` message.

Here is the README, I'll add comments italicized:

1. Install the plugin with `ddev get valthebald/ddev-matomo` ✅
2. Review your database settings in .ddev/docker-compose.matomo.yaml
    - _**^^ Do I need to do something here?**_
4. Create matomo database if needed with `echo "SHOW TABLES;" | ddev import-db --database=${MATOMO_DATABASE_DBNAME}`'
    - _**^^ I am not sure what this does ...**_
6. Restart DDEV to install it with `ddev restart` ✅
7. Access https://matomo.\<projectname\>.ddev.site in your browser and follow the installation steps.
    - _**^^ Here I get the error `Could not connect to the database server: Database access denied`**_

**Comment by gitressa** [comment]
I would love to get this to work.

Wouldn't it be awesome to get it included as an official [DDEV add-on](https://ddev.readthedocs.io/en/latest/users/extend/additional-services/#creating-an-additional-service-for-ddev-add-on)?

**Comment by madsnorgaard** [comment]
I am running into the same issue on my side.

Followed the steps and got the same issue as you did. However when I went to do step 2 I set the following which made the install go through:

`MATOMO_DATABASE_TABLES_PREFIX: "matomo"`

I believe the issue is that the MAtomo installation uses the existing db container and might require prefix for that very reason.

**Comment by madsnorgaard** [comment]
I agree this would be a great addition to DDEV but it seems to clash with Drupal when running this in DDEV already.

**Comment by alfaguru** [comment]
I was able to proceed by using the root/root credentials. It appears that db/db doesn't have sufficient permissions to create the database.

**Comment by lramarojaona** [comment]
You might want to try this one instead: https://github.com/madsnorgaard/ddev-matomo
